doc/man3/flux_request_decode.adoc

```diff
@@ -22,7 +22,7 @@ SYNOPSIS
 
  int flux_request_decode_raw (const flux_msg_t *msg,
                               const char **topic,
-                              void *data, int *len);
+                              void **data, int *len);
```

Comment:
OK, sounds like @grondo already acked so let's fix this.  Want to go ahead and also fix:
```c
int flux_msg_get_payload (const flux_msg_t *msg, int *flags,
                          void *buf, int *size);
int flux_mrpc_get_raw (flux_mrpc_t *mrpc, void *data, int *len);
int flux_rpc_get_raw (flux_future_t *f, void *data, int *len);
int flux_response_decode_raw (const flux_msg_t *msg, const char **topic,
                              void *data, int *len);
```
for consistency?